cg/cli/set/base.py

```diff
@@ -123,43 +123,42 @@ def _get_samples(
 
 def _get_samples_by_identifiers(
```

Comment:
is this duplicate of cg/cli/set/families?

Reply:
Out of scope!!! 😄

Reply:
No but seriously I don't dare to mess more with stuff in this PR